opsdroid/connector/matrix/connector.py

```diff
@@ -210,6 +210,13 @@ def filter_json(self):
             except Exception:  # pylint: disable=W0703
                 _LOGGER.exception(_("Matrix sync error."))
 
+    def lookup_target(self, alias):
+        """Convert a room alias to the corresponding room ID."""
```

Comment:
Can this also lookup the config names for these rooms? i.e. `'main'`? If so the docstring is a little misleading, and if not it would be cool if it could. As that allows skill developers to have different rooms which the user can then map to actual matrix rooms / aliases.